cli/utils.js

```diff
@@ -68,14 +68,29 @@ const getGraph = (issue, prefix, showFullManifest) => {
         issue.from.map(({ project }) => project)
     );
     const isMultipleBranches = uniqueProjectNamePrefixes.size > 1;
-    return issue.from
-        .map(({ project, paths }) => {
-            const root = getManifestName(
-                project,
-                isMultipleBranches || showFullManifest
-            );
-            return `${prefix}${root} > ${paths.join(' > ')}`;
-        })
+    const pathsMap = issue.from.reduce((acc, { project, paths }) => {
+        const root = getManifestName(
+            project,
+            isMultipleBranches || showFullManifest
+        );
+        let path;
+        if (issue.from.length > 20 && paths.length > 1) {
+            // If there are lots of paths, reduce those that are transitive dependencies
+            path = `${paths[0]} > ... > ${paths[paths.length - 1]}`;
+        } else {
+            path = paths.join(' > ');
+        }
+        const key = `${prefix}${root} > ${path}`;
+        const val = acc.get(key);
+        if (val) {
+            acc.set(key, val + 1);
+        } else {
+            acc.set(key, 1);
+        }
+        return acc;
+    }, new Map());
+    return Array.from(pathsMap.entries())
+        .map(([k, v]) => (v === 1 ? k : k.replace('...', `... (x${v})`)))
         .join('\r\n');
```

Comment:
When creating a batch issue:

If there are lots of paths (more than 20) for a vulnerable transitive dependency, for example `z@1.0.0`:
```
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > v@1.0.0 > w@1.0.0 > x@1.0.0 > y@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > v@1.0.0 > w@1.0.0 > x@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > v@1.0.0 > w@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > v@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > z@1.0.0
project > a@1.0.0 > z@1.0.0
```

That will be reduced to one line:
```
project > a@1.0.0 > ... (x25) > z@1.0.0
```